mackup/applications/autojump.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = Autojump
+
+[configuration_files]
+Library/autojump
```

Comment:
new line missing.
There are log files in this folder, please only sync config files.